warp/src/tesseract/mod.rs

```diff
@@ -534,52 +525,11 @@ impl Tesseract {
 
 /// Methods for wasm only
 #[cfg(target_arch = "wasm32")]
-#[cfg_attr(target_arch = "wasm32", wasm_bindgen)]
 impl Tesseract {
-    #[wasm_bindgen(js_name = set)]
-    pub fn set_wasm(&self, key: &str, value: &str) -> std::result::Result<(), JsError> {
-        self.set(key, value).map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = retrieve)]
-    pub fn retrieve_wasm(&self, key: &str) -> std::result::Result<String, JsError> {
-        self.retrieve(key).map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = update_unlock)]
-    pub fn update_unlock_wasm(
-        &self,
-        old_passphrase: &[u8],
-        new_passphrase: &[u8],
-    ) -> std::result::Result<(), JsError> {
-        self.update_unlock(old_passphrase, new_passphrase)
-            .map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = delete)]
-    pub fn delete_wasm(&self, key: &str) -> std::result::Result<(), JsError> {
-        self.delete(key).map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = unlock)]
-    pub fn unlock_wasm(&self, passphrase: &[u8]) -> std::result::Result<(), JsError> {
-        self.unlock(passphrase).map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = save)]
-    pub fn save_wasm(&self) -> std::result::Result<(), JsError> {
-        self.save().map_err(|e| e.into())
-    }
-
-    #[wasm_bindgen(js_name = subscribe)]
-    pub fn subscribe_wasm(&self) -> AsyncIterator {
-        AsyncIterator::new(Box::pin(self.subscribe().map(|t| Into::<JsValue>::into(t))))
-    }
-
     /// Used to load contents from local storage
-    pub fn load_from_storage(&self) -> std::result::Result<(), JsError> {
+    pub fn load_from_storage(&self) -> std::result::Result<(), Error> {
```

Comment:
I wonder if we should move this function in whole to `warp-wasm` repo. Thoughts?

Reply:
Would have been nice, but `load_from_storage` accesses private stuff from InnerTesseract, so we wouldn't be able to copy it out as is. But perhaps in a future PR we could rewrite it to achieve the same behavior via other public exported methods?